tests/unittest_brain.py

```diff
@@ -1659,6 +1659,34 @@ def test_typing_types(self) -> None:
             inferred = next(node.infer())
             self.assertIsInstance(inferred, nodes.ClassDef, node.as_string())
 
+    def test_typing_newtype_attrs(self) -> None:
+        ast_nodes = builder.extract_node(
+            """
+        from typing import List, NewType
+        import typing
+        NewType("Foo", str) #@
+        NewType("Bar", "int") #@
+        """
+        )
+        assert isinstance(ast_nodes, list)
+
+        # Base type given by reference
+        foo_node = ast_nodes[0]
+        foo_inferred = next(foo_node.infer())
+        self.assertIsInstance(foo_inferred, astroid.ClassDef)
+
+        # Check base type method is inferred
+        foo_base_class_method = foo_inferred.getattr("endswith")[0]
+        self.assertIsInstance(foo_base_class_method, astroid.FunctionDef)
```

Comment:
I'm not sure about this, but is there a way to infer that `endswith` comes from `str`? Some attribute within the `FunctionDef` node or the `qname()` of the node?
If there is I think it would be good to check this as well.

Reply:
@colatkinson Did you consider this comment?

Reply:
Yup, I ended up just going with the `qname()` check.

Reply:
Ah sorry! I missed that, thanks!